staging/src/k8s.io/apimachinery/pkg/util/net/util_test.go

```diff
@@ -66,3 +69,28 @@ func TestIPNetEqual(t *testing.T) {
 		}
 	}
 }
+
+func TestIsConnectionRefused(t *testing.T) {
+	testCases := []struct {
+		err    error
+		expect bool
+	}{
+		{
+			&url.Error{Err: &net.OpError{Err: syscall.ECONNRESET}},
+			false,
+		},
+		{
+			&url.Error{Err: &net.OpError{Err: syscall.ECONNREFUSED}},
+			true,
+		},
+		{&url.Error{Err: &net.OpError{Err: &os.SyscallError{Err: syscall.ECONNREFUSED}}},
```

Comment:
the formatting above looks weird. please fix.

Reply:
> the formatting above looks weird. please fix.

The format above is same with the format of the test cases which already exists in the same package.